lib/graphql/schema.ts

```diff
@@ -9,6 +9,7 @@ export default `
     applications(filter: ApplicationsFilter): QueryApplicationsResult
     application(id: ID!): Application
     permits: [Permit!]
+    generateApplicantsReport(input: GenerateApplicantsReportInput!): GenerateApplicantsReportResult!
```

Comment:
The notion ticket said that generateApplicantsReport should be a mutation but I think it makes more sense as a query since no data was modified. lmk if it should be changed to a mutation instead

Reply:
That makes sense. Good catch.

Reply:
```suggestion
    generateApplicantsReport(input: GenerateApplicantsReportInput!): GenerateApplicantsReportResult
```
Needs to be nullable if authorization fails